common-app-lib/src/cmr/common_app/services/kms_fetcher.clj

```diff
@@ -39,6 +39,7 @@
    :temporal-keywords [:temporal-resolution-range]
    :location-keywords [:category :type :subregion-1 :subregion-2 :subregion-3]
    :science-keywords [:category :topic :term :variable-level-1 :variable-level-2 :variable-level-3]
+   :passes [:pass]
```

Comment:
Would this break hierarchical keywords? `passes` is not really a valid KMS keyword.